internal/move2kubeapi/move2kubeapi.go

```diff
@@ -135,20 +134,25 @@ func getAssetsList(w http.ResponseWriter, r *http.Request, isCustomization bool)
 	name := mux.Vars(r)["name"]
 	assets, err := m2kapp.GetAssetsList(name, isCustomization)
 	if err != nil || assets == nil {
-		log.Errorf("Could not get the assets for the app %s . Error: %q", name, err)
+		log.Errorf("Could not get the assets/customizations for the app %s . Error: %q", name, err)
 		w.WriteHeader(http.StatusNotFound)
 		return
 	}
 	w.WriteHeader(http.StatusOK)
 	w.Header().Set("Content-Type", "application/json")
 	if err := json.NewEncoder(w).Encode(assets); err != nil {
-		log.Errorf("Error while getting assets list : %s", err)
+		log.Errorf("failed to marhsal the assets/customizations list %+v to json. Error: %q", assets, err)
 	}
 }
 
 func getAsset(w http.ResponseWriter, r *http.Request, isCustomization bool) {
 	name := mux.Vars(r)["name"]
-	asset := mux.Vars(r)["asset"]
+	asset := ""
+	if isCustomization {
+		asset = mux.Vars(r)["customization"]
```

Comment:
Do we want to keep the flag name same as in command line? "customization**s**"

Reply:
it is the same, if you look at the url endpoints I have tried to keep it analogous to the `assets`
https://github.com/konveyor/move2kube-api/blob/main/internal/move2kubeapi/move2kubeapi.go#L388-L396